backend/src/main/resources/db/migration/V3__create_favourite_activity.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS spar.favourite_activity (
+  id          SERIAL,
+  user_id     VARCHAR(70) NOT NULL,
```

Comment:
Shouldn't we have a primary key here? Even if we think we don't _need_ one now, I'm of the opinion that removing a constraint in the future goes a lot smoother than adding one.

Reply:
The PK is in the line 8.

Reply:
Sorry, I meant to say _foreign_ key. But, like I wrote earlier, I believe we can go without it for now.